<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaserControl</class>
 <widget class="QDialog" name="LaserControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>224</width>
    <height>168</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Laser Settings</string>
  </property>
  <widget class="QPushButton" name="BTN_CONTROL_A_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_CONTROL_A_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_CONTROL_B_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_CONTROL_B_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_CONTROL_C_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_CONTROL_C_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_CONTROL_D_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_CONTROL_D_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="SLIDER_SPEED_1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>44</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="LASER_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>44</y>
     <width>32</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string># 1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EDT_SPEED_1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>24</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="EDT_SPEED_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>24</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="SLIDER_SPEED_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>74</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="LASER_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>74</y>
     <width>32</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string># 2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EDT_SPEED_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>24</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="SLIDER_SPEED_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>104</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="LASER_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>104</y>
     <width>32</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string># 3</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EDT_SPEED_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>24</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="SLIDER_SPEED_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>134</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="LASER_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>134</y>
     <width>32</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string># 4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>10</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>LASER RADIUS</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
